waiter/src/waiter/core.clj

```diff
@@ -148,10 +148,17 @@
                                "/owners" :token-owners-handler-fn
                                "/refresh" :token-refresh-handler-fn
                                "/reindex" :token-reindex-handler-fn}
-                     "waiter-async" {["/complete/" :request-id "/" :service-id] :async-complete-handler-fn
+                     "waiter-async" {;; async api version 1
```

Comment:
We should update the unit tests in `waiter.core-test/test-waiter-request?-factory` , `waiter.core-test/test-routes-mapper` and `waiter.core-test/test-async-*`

Reply:
Done